source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst

```diff
@@ -56,6 +56,11 @@ format is::
 
     mongodb+srv://{hostname}.{domainname}/{options}
```

Comment:
nitpick: isn't it enough to just say `mongodb+srv://{hostname}.{domainname}/{db}?{options}`, then you don't need to clarify below, and can refer to it directly as `db` in the tests?

Reply:
I've merged it without changing this, as it does add clarity to have it separate, and you wouldn't have to explain that ? is optional etc.